src/orion/core/worker/experiment.py

```diff
@@ -385,7 +388,7 @@ def stats(self):
             )
         completed_trials = self._db.read('trials', query,
                                          selection={'_id': 1, 'end_time': 1,
-                                                    'results': 1})
+                                                    'results': 1, 'experiment': 1, 'params': 1})
```

Comment:
Why is that?

Reply:
Because `trial.id` is built using params and experiment's id. So if we query the db and build the trials using an incomplete definition, we can't recompute `trial.id`.

Reply:
This would raise an error without the `experiment` and `params` when getting the id since we don't have the experience or the params set to a value.

Reply:
@DatCorno You can drop the _id in the selection.

Reply:
@DatCorno Second change is here.